js/gulpfile.js

```diff
@@ -8,6 +8,20 @@ function exec(command, cb) {
 
 var protoc = process.env.PROTOC || '../src/protoc';
 
+var wellKnownTypes = [
```

Comment:
What about `google/protobuf/compiler/plugin.proto`? It looks like that's also on the list of well known types.